zetaclient/chains/bitcoin/observer/inbound.go

```diff
@@ -265,6 +265,14 @@ func (ob *Observer) NewInboundVoteFromStdMemo(
 	event *BTCInboundEvent,
 	amountSats *big.Int,
 ) *crosschaintypes.MsgVoteInbound {
+	// determine amount and coin type according to operation code
+	// NoAssetCall expects no asset transfer; any excess amount above depositor fee is ignored
+	coinType := coin.CoinType_Gas
+	if event.MemoStd.OpCode == memo.OpCodeCall {
+		amountSats = big.NewInt(0)
```

Comment:
So the sender sends any amount to the TSS and this amount is ignored like a donation?

I think we should introduce a max limit for no asset call to not end up in a situation where a large amount is sent in inadvertence and lost for the sender

Reply:
okay, we can try setting a max for remaining amount to revert to the sender.